src/main/cs/Infinispan/HotRod/Impl/RemoteCacheSWIGImpl.cs

```diff
@@ -253,6 +261,75 @@ public byte[] Execute(string scriptName, IDictionary<string, string> dict)
             return ret;
         }
 
+        public Task<V> GetAsync(K key)
+        {
+            return Task.Run(() => Get(key));
+        }
+
+        public Task<V> PutAsync(K key, V val, ulong lifespan = 0, TimeUnit lifespanUnit = TimeUnit.SECONDS, ulong maxIdleTime = 0, TimeUnit maxIdleUnit = TimeUnit.SECONDS)
```

Comment:
I believe GetAsync and PutAsync methods should have "async" modifier and should "await" for the Task.Run? ...."The marked async method can itself be awaited by methods that call it."